systemv/lpstat.c

```diff
@@ -660,7 +660,7 @@ match_list(const char *list,		/* I - List of names */
 
     while (*list && !isspace(*list & 255) && *list != ',')
       list ++;
-  }
+  } while (*list);
```

Comment:
Minor coding style nit - the "while" should be on a separate line from the closing brace.